app/models/concerns/spotlight/ar_light.rb

```diff
@@ -53,6 +53,10 @@ def generated_association_methods
           mod
         end
       end
+
+      def validators_on *args
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `validators_on(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.